packaging/nuget/package.ps1

```diff
@@ -0,0 +1,26 @@
+Push-Location $PSScriptRoot
+[Environment]::CurrentDirectory = $PWD
```

Comment:
Also, changing the current directory is generally bad practice in a PowerShell script here. You'll need to wrap the whole thing in `try..finally` to ensure that if any thing fails in the script, the changes are reverted. I'd just use paths relative to either `$PSScriptRoot` or use `_common.ps1` and `$RepoRoot` everywhere.